net-im/synapse/synapse-1.74.0-r1.ebuild

```diff
@@ -127,6 +127,7 @@ RDEPEND="${DEPEND}
 	systemd? ( dev-python/python-systemd[${PYTHON_USEDEP}] )
 "
 BDEPEND="
+	<dev-python/poetry-core-1.5.0[${PYTHON_USEDEP}]
```

Comment:
It doesn't matter for now as we're on it & aware of the issue but fwiw, I usually leave a comment above < deps with a link to a bug so people know when it can be dropped. But it's not necessary to spin another version of this PR.

Reply:
I was thinking about it but I decided to leave it as is becuase I expect it to be resolved soon.

Reply:
No problem.